corehq/apps/geospatial/templates/gps_capture.html

```diff
@@ -0,0 +1,167 @@
+            <h3 class="panel-title">
+                {% if data_type == 'user' %}
+                    {% trans 'Users without GPS Data' %}
+                {% else %}
+                    {% trans 'Cases without GPS Data' %}
+                {% endif %}
+            </h3>
+        </div>
+        <div class="panel-body">
+            <div class="row">
+                <div class="col-sm-6">
+                    <search-box data-apply-bindings="false"
+                                params="value: query,
+                                        action: function() { goToPage(1); },
+                                        placeholder: '{% trans_html_attr "Search..." %}'"></search-box>
+                </div>
+            </div>
+            <table class="table table-striped table-responsive"
+                   style="margin-botton: 0;"
+                   data-bind="visible: showTable">
+                <thead>
+                    <tr>
+                        <th class="col-xs-3">
+                            {% if data_type == 'user' %}
+                                {% trans "Username" %}
+                            {% else %}
+                                {% trans "Name" %}
+                            {% endif %}
+                        </th>
+                        <th class="col-xs-3">{% trans "Latitude" %}</th>
+                        <th class="col-xs-3">{% trans "Longitude" %}</th>
+                        <th class="col-xs-3">{% trans "Actions" %}</th>
+                    </tr>
+                </thead>
+                <tbody data-bind="foreach: dataItems">
+                    <tr>
+                        <td>
+                            <a target="_blank" data-bind="attr: {'href': url}">
+                                <span data-bind="text: name"></span>
+                            </a>
+                        </td>
+                        <td>
+                            <div data-bind="css: {'has-error': !isLatValid()}">
+                                <input class="form-control"
+                                       type="text"
+                                       name="latInput"
+                                       placeholder="{% trans 'Enter latitude...' %}"
+                                       data-bind="value: lat, event: {change: onValueChanged}" />
+                                <span class="help-block" data-bind="visible: !isLatValid() && !isNaN(lat())">
+                                    {% blocktrans %}
+                                        The value should be between -90 and 90
+                                    {% endblocktrans %}
+                                </span>
+                                <span class="help-block" data-bind="visible: !isLatValid() && isNaN(lat())">
+                                    {% blocktrans %}
+                                       The value must be a number
+                                    {% endblocktrans %}
```

Comment:
I believe you could simply combine the two to "The value must be a number between -90 and 90".
Splitting this into two separate errors seems like an overkill.
Same for the longitude below.

Reply:
If you're updating this Zandre, lines 108 - 118 should then also be updated :smiley:

Reply:
[0e24392](https://github.com/dimagi/commcare-hq/pull/33420/commits/0e24392b6c5155d5fd0bff69abad2acb0fa43642)